src/ProductCard/ProductCardDescription/index.tsx

```diff
@@ -7,20 +7,22 @@ export type ProductCardDescriptionProps = {
   description: string;
   buttonText?: string;
   url: string;
+  isButtonActive?: boolean;
 };
 
 const ProductCardDescription: React.FC<ProductCardDescriptionProps> = ({
   productName,
   description,
   buttonText,
-  url
+  url,
+  isButtonActive = true
 }) => (
   <div className={styles['product-card-description-box']}>
     <div className={styles['product-card-name']}>{productName}</div>
     <p className={styles['product-card-description']}>{description}</p>
     {buttonText &&
       <div className={styles['row']}>
-        <a className={styles['buy-button']} href={url}>
+        <a className={isButtonActive ? styles['buy-button'] : styles['buy-button-disabled']} href={isButtonActive ? url : "#"}>
```

Comment:
how about also putting a `disabled` attribute on the link? you could also change the name of this flag to `disabled`

Reply:
also please correct the spelling of "hackoctoberfest" -> 
<img width="144" alt="image" src="https://user-images.githubusercontent.com/10517315/194603413-144f0a2a-30af-48e4-b70f-3f0f1b9b8d38.png">

Reply:
ok, but the spelling is correct